src/Avalonia.Diagnostics/Diagnostics/Controls/FilterTextBox.axaml

```diff
@@ -27,7 +27,9 @@
             <Image>
               <DrawingImage>
                 <GeometryDrawing Brush="{Binding $parent[ToggleButton].Foreground}">
-                  M7.495 9.052L8.386 11.402H9.477L6.237 3H5.217L2 11.402H3.095L3.933 9.052H7.495ZM5.811 4.453L5.855 4.588L7.173 8.162H4.255L5.562 4.588L5.606 4.453L5.644 4.297L5.676 4.145L5.697 4.019H5.72L5.744 4.145L5.773 4.297L5.811 4.453ZM13.795 10.464V11.4H14.755V7.498C14.755 6.779 14.575 6.226 14.216 5.837C13.857 5.448 13.327 5.254 12.628 5.254C12.429 5.254 12.227 5.273 12.022 5.31C11.817 5.347 11.622 5.394 11.439 5.451C11.256 5.508 11.091 5.569 10.944 5.636C10.797 5.703 10.683 5.765 10.601 5.824V6.808C10.867 6.578 11.167 6.397 11.505 6.268C11.843 6.139 12.194 6.075 12.557 6.075C12.745 6.075 12.915 6.103 13.07 6.16C13.225 6.217 13.357 6.306 13.466 6.427C13.575 6.548 13.659 6.706 13.718 6.899C13.777 7.092 13.806 7.326 13.806 7.599L11.995 7.851C11.651 7.898 11.355 7.977 11.107 8.088C10.859 8.199 10.654 8.339 10.492 8.507C10.33 8.675 10.21 8.868 10.132 9.087C10.054 9.306 10.015 9.546 10.015 9.808C10.015 10.054 10.057 10.283 10.139 10.496C10.221 10.709 10.342 10.893 10.502 11.047C10.662 11.201 10.862 11.323 11.1 11.413C11.338 11.503 11.613 11.548 11.926 11.548C12.328 11.548 12.686 11.456 13.001 11.27C13.316 11.084 13.573 10.816 13.772 10.464H13.795ZM11.667 8.721C11.843 8.657 12.068 8.607 12.341 8.572L13.806 8.367V8.976C13.806 9.222 13.765 9.451 13.683 9.664C13.601 9.877 13.486 10.063 13.34 10.221C13.194 10.379 13.019 10.503 12.816 10.593C12.613 10.683 12.39 10.728 12.148 10.728C11.961 10.728 11.795 10.703 11.653 10.652C11.511 10.601 11.392 10.53 11.296 10.441C11.2 10.352 11.127 10.247 11.076 10.125C11.025 10.003 11 9.873 11 9.732C11 9.568 11.018 9.421 11.055 9.292C11.092 9.163 11.16 9.051 11.257 8.958C11.354 8.865 11.491 8.785 11.667 8.721Z
+                    <GeometryDrawing.Geometry>
```

Comment:
But why it is reported as an error?

Reply:
Not sure. It does have the `ContentAttribute` set, so it seems like it should work... Searched the code base for other usages where the Geometry is set using content, but couldn't find any.
This is the error presented in Rider:
```
  <Avalonia.Diagnostics>\Diagnostics\Controls\FilterTextBox.axaml:1247 Text is not expected here
```
and visual studio:
![image](https://user-images.githubusercontent.com/807383/120931420-b0c43a00-c6f1-11eb-98c2-f9011fdb8456.png)

Reply:
It feels like it should be reported to JetBrains.
Though I am fine with change in this PR anyway.

Reply:
Maybe... I'm not sure how the (A)XAML-parsing works inside the IDEs. As I mentioned above, this happens in Visual Studio as well. Unlike WPF/UWP errors though, the errors cannot be found inside the Error List / Issue count:
![image](https://user-images.githubusercontent.com/807383/121002952-3812bb80-c78d-11eb-8758-dbfb8762bba6.png)